src/main/resources/application.properties

```diff
@@ -1,6 +1,6 @@
 spring.datasource.driver-class-name=org.postgresql.Driver
-spring.datasource.url=jdbc:postgresql://localhost:5432/postgres
-spring.datasource.username=postgres
+spring.datasource.url=jdbc:postgresql://localhost:5432/ogloszenia
```

Comment:
Shouldn't we be using eng names? Like `ads` or z`helpoffers`

Reply:
will do